packages/markers/src/browser/problem/problem-composite-tree-node.ts

```diff
@@ -0,0 +1,84 @@
+/********************************************************************************
+ * Copyright (C) 2021 Ericsson and others.
```

Comment:
@LukasBoll you would either put your company name, or actual name. I don't believe you work at Ericsson?